src/test/java/com/wikia/webdriver/testcases/mercurytests/SmartBannerTests.java

```diff
@@ -22,14 +22,19 @@
   public void prepareTest() {
     driver.manage().timeouts().pageLoadTimeout(100, TimeUnit.SECONDS);
   }
-  
-  private static final String[]
-      DIFFERENT_HUBS_WIKIS =
-      {"destiny", "cocktails", "thehungergames", "dc", "tardis", "starwars", "lego"};
-  private static final String[]
-      DIFFERENT_HUBS_COLORS =
-      {"#94d11f", "#8ca038", "#ff7f26", "#ff5400", "#00b7e0", "#09d3bf",
-       "#ffd000"};
+
+  /**
+   * Wiki name, Main page, HUB color
+   */
+  private static final String[][] WIKIS = {
+      {"destiny", "Destiny_Wiki", "#94d11f"},
+      {"cocktails", "Cocktails_Wiki", "#8ca038"},
+      {"thehungergames", "The_Hunger_Games_Wiki", "#ff7f26"},
+      {"dc", "Main_Page", "#ff5400"},
+      {"tardis", "Doctor_Who_Wiki", "#00b7e0"},
+      {"starwars", "Main_Page", "#09d3bf"},
+      {"lego", "LEGO_Wiki", "#ffd000"}
+  };
```

Comment:
Sometimes i will need two or more packages in the same test that is why i cannot use DataProvider. From the other hand in MercuryArticles.java, i'm calling only few of all articles in the same test using static fields.